docs/content/guides/columns/column-filter.md

```diff
@@ -155,765 +449,3044 @@ import 'handsontable/dist/handsontable.full.min.css';
+          price: 130.1,
+          sellDate: 'Mar 27, 2023',
+          sellTime: '03:17 AM',
+          inStock: true,
+        },
+        {
+          brand: 'Chatterpoint',
+          model: 'Road Tire Tube',
+          price: 59,
+          sellDate: 'Aug 28, 2023',
+          sellTime: '08:01 AM',
+          inStock: true,
+        },
+        {
+          brand: 'Eidel',
+          model: 'HL Road Tire',
+          price: 279.99,
+          sellDate: 'Oct 2, 2023',
+          sellTime: '01:23 AM',
+          inStock: true,
+        },
+      ]}
+      columns={[
+        {
+          title: 'Brand',
+          type: 'text',
+          data: 'brand',
+        },
+        {
+          title: 'Model',
+          type: 'text',
+          data: 'model',
+        },
+        {
+          title: 'Price',
+          type: 'numeric',
+          data: 'price',
+          numericFormat: {
+            pattern: '$ 0,0.00',
+            culture: 'en-US',
+          },
+        },
+        {
+          title: 'Date',
+          type: 'date',
+          data: 'sellDate',
+          dateFormat: 'MMM D, YYYY',
+          correctFormat: true,
+          className: 'htRight',
+        },
+        {
+          title: 'Time',
+          type: 'time',
+          data: 'sellTime',
+          timeFormat: 'hh:mm A',
+          correctFormat: true,
+          className: 'htRight',
+        },
+        {
+          title: 'In stock',
+          type: 'checkbox',
+          data: 'inStock',
+          className: 'htCenter',
+        },
+      ]}
+      // enable filtering
+      filters={true}
+      // enable the column menu
+      dropdownMenu={true}
+      // to differentiate this example's CSS from other examples on this page
+      className="customFilterButtonExample2"
+      height="auto"
+      licenseKey="non-commercial-and-evaluation"
+    />
+  );
+};
+
+/* start:skip-in-preview */
+ReactDOM.render(<App />, document.getElementById('exampleCustomFilterButton2'));
+/* end:skip-in-preview */
+```
+
+```css
+/* hide the column menu button by default */
+.customFilterButtonExample2 .changeType {
+  visibility: hidden;
+}
+
+/* show the column menu button on hover */
+.customFilterButtonExample2 .relative:hover .changeType {
+  visibility: visible;
+}
+```
+
+:::
+
+:::
+
+## Change the width of the filter menu
+
+If the text data in your columns is too long to fit in the filters container, you can adjust the
+column menu's width for better user experience. You can achieve this with by styling
+`.htDropdownMenu table.htCore`.
+
+```css
+.handsontable .htDropdownMenu table.htCore {
+  width: 300px !important;
+}
+```
+
+## Exclude rows from filtering
+
+You can exclude any number of top or bottom rows from filtering.
+
+In the following demo, the first and the last row are [frozen](@/guides/rows/row-freezing.md), and
+filtering doesn't affect them.
+
+::: only-for javascript
+
+::: example #exampleExcludeRowsFromFiltering --html 1 --js 2
```

Comment:
The bottom frozen row doesn't make sense here. Let's at least add 'minSpareRows: 1' there. https://jsfiddle.net/zmg8r0yp/1/